README.md

```diff
@@ -22,7 +22,42 @@ This gem supports the following versions of Ruby and Rails:
 gem 'response_bank'
 ```
 
-2. use `#response_cache` method to any desired controller's action
+2. add an initializer file. We need to configure the `acquire_lock` method, set the cache store and the logger
+
+```ruby
+require 'response_bank'
+
+module ResponseBank
+  LOCK_TTL = 90
+
+  class << self
+    def acquire_lock(cache_key)
```

Comment:
acquiring & releasing cache as example usage is out of scope I think

Reply:
I know that the gem requires us to [override the `acquire_lock` method](https://github.com/Shopify/response_bank/blob/0c9ce646aad818ca15e93ba119fac8d5a426ee22/lib/response_bank.rb#L17), which is why I added it to the README.

Reply:
oh  I just knew about it
I've been using this gem on production for 1 year without any issues